frontend/public/components/row-filter.jsx

```diff
@@ -22,6 +22,8 @@ class CheckBoxes_ extends React.Component {
   constructor(props) {
     super(props);
     this.state = {selected: []};
+    this.countNumItems = this.countNumItems.bind(this);
```

Comment:
You don't need to bind to this since it's not used as a callback / event handler.